### ISSUE TITLE
Improvements needed

### ISSUE THREAD

**Issue by ChrisLeeGit** [scrapy-plugins/scrapy-deltafetch#13]
This plugin is very helpful to me, thanks. 

Some improvements needed:
-  using `logging` module instead.
- `db.has_key(key)` is deprecated in `bsddb3`, using `key in db` instead.

**Comment by redapple** [comment]
Hi @ChrisLeeGit, thanks for your feedback
* I'm seeing now that we haven't released a version with `logging` module (which is in the master branch). will work on this
* I'll have a look at bsddb3 `has_key` thing

**Comment by redapple** [comment]
@ChrisLeeGit , can you point me to the deprecation note on `db.has_key(key)`?

**Comment by ChrisLeeGit** [comment]
In PyCharm, it hints that "PEP8: `.has_key()` is deprecated, use `in`". BTW, I use Python 3.4.3.
```
def process_spider_output(self, response, result, spider):
        for r in result:
            if isinstance(r, Request):
                key = self._get_key(r)
                if self.db.has_key(key):
                    logger.info("Ignoring already visited: %s" % r)
                    if self.stats:
                        self.stats.inc_value('deltafetch/skipped', spider=spider)
                    continue
```

**Comment by redapple** [comment]
@ChrisLeeGit , fyi, I've just released v1.2.0 with `logging` logging

**Comment by redapple** [comment]
@ChrisLeeGit , alright. I'll have a look
